packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -311,9 +313,9 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           }
         ) }
         onClick={ () => this._onItemClick(item.index) }
-        role='menu'
+        role='option'
         aria-selected={ this.state.selectedIndex === item.index ? 'true' : 'false' }
-        aria-label={ item.text }
+        ariaLabel={ item.text }
```

Comment:
I created https://github.com/OfficeDev/office-ui-fabric-react/issues/1621 for this "issue"